### ISSUE TITLE
bug with permanent raised shadows

### ISSUE THREAD

**Issue by Zamiell** [Zamiell/hanabi-live#1736]
step 1 - start a new game
step 2 - play a card from your hand
step 3 - go into the in-game replay
step 4 - click and hold on the you played, which will draw the shadow
step 5 - press the right arrow key so that the card plays on to the stacks

now, the card will be bugged forever, with the shadow permanently drawn

this functionality is defined here:
https://github.com/Zamiell/hanabi-live/blob/master/client/src/game/ui/HanabiCard.ts#L825-L871

the fix is to (probably) add more shit to the appropriate observer in StateObserver.ts

**Comment by DrAnax** [comment]
Can't reproduce it reliably, I tried the following:

1. start 2p game, locally
2. play 1-2 playable cards
3. toggle in-game replay, try steps 4 & 5, nothing happened, no shadows drawn (cards flip and show empathy when click-holded)
4. play / discard a few more cards, retry above step (3), nothing happened
5. end the game, go where the cards were played, try above step (3), nothing happened.

I did bug a card, as mentioned, during a hypo. That's the only case I could find where a click-holded card is raised. There's a bigger than normal shadow drawn, which stays on screen and follows the card. Is that it?
